code/script.js

```diff
@@ -0,0 +1,79 @@
+const userSection = document.getElementById ('userSection')
+const repoSection = document.getElementById ('repoSection')
+
+const username = 'mistscale'
+const API_USER_URL = `https://api.github.com/users/${username}`
+const API_REPOS_URL = `https://api.github.com/users/${username}/repos`
+
+const options = {
+    method: 'GET',
+    headers: {
+        Authorization: 'API_TOKEN'
+      }
+  }
+
+// Display profile picture and username from user api
+const getUser = () =>  {
+    fetch(API_USER_URL, options)
+    .then(res => res.json())
+    .then(data => {
+        userSection.innerHTML = `
+        <img src='${data.avatar_url}' class='profile-img'/>
+        <h1>${data.login}</h1>`
+    })
+   getRepos() 
+}
+
+const getRepos = () => {
+    fetch(API_REPOS_URL, options)
+    .then(res => res.json())
+    .then(data => {
+       const forkedRepos = data.filter((repo) => repo.fork && repo.name.startsWith('project'))
+       forkedRepos.forEach((repo) => {
+           repoSection.innerHTML += `
+           <div class='repo-name' id='${repo.name}'>
+            <h2>${repo.name}</h2>
+            <p>Repo link: <a href='${repo.html_url}'>here</a></p>
+            <p>Default branch: ${repo.default_branch}</p>
+            <p>Recent update: ${new Date(repo.pushed_at).toLocaleDateString('sv-SE')}</p>
+           </div>`
+       })
+       getPullRequests(forkedRepos)
+       activateChart(forkedRepos.length)
+    })
```

Comment:
Love how you keep each function so elegant and use it to call the next functions. Your javascript is really gorgeous to read. You get precisely what you need and pass it on to the next functions. In my opinion, your javascript is a great example of perfect use of occam's razor (or the law of parsimony, the problem-solving principle that entities should not be multiplied beyond necessity).